src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,243 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createCarQuery = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement saveCarStatement = connection.prepareStatement(createCarQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            saveCarStatement.setLong(1, car.getManufacturer().getId());
+            saveCarStatement.setString(2, car.getModel());
+            saveCarStatement.executeUpdate();
+            ResultSet resultSet = saveCarStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't create " + car + ". ", throwable);
+        }
+        createCarsAndDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Car get(Long id) {
+        String query = "SELECT "
+                + "cars.id, "
+                + "cars.model, "
```

Comment:
```suggestion
        String query = "SELECT cars.id, cars.model, "
```
ideally I would form queries like this